joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -69,6 +69,7 @@ JointTrajectoryController::init(
   // with the lifecycle node being initialized, we can declare parameters
   lifecycle_node_->declare_parameter<std::vector<std::string>>("joints", joint_names_);
   lifecycle_node_->declare_parameter<std::vector<std::string>>("write_op_modes", write_op_names_);
+  lifecycle_node_->declare_parameter<double>("state_publish_rate", 50.0);
```

Comment:
why do you think an update rate of `50hz` for the current state is needed?
Thinking forward, the preferred way of triggering a new trajectory is by calling an action service, which has in built feedback mechanism. Wouldn't that suffice?

Reply:
feedback only occurs when there is a current action goal. state publishing publishes despite there being no action goal.

50hz was the previous setting in ros_controllers.